habitat_sim/simulator.py

```diff
@@ -147,9 +156,13 @@ def semantic_scene(self):
         return self._sim.semantic_scene
 
     def get_sensor_observations(self):
+        for _, sensor in self._sensors.items():
```

Comment:
Since this PR uses as many framebuffers as there are sensors, the draw calls for each sensor are now independent.  Breaking the the draw and read into two separate blocks allows OpenGL to do a better job scheduling them.